### PR TITLE
Canvaspage

### PR DESCRIPTION
**Pull request by DNaunin**
Created the Page where you can combine fabric and pattern, included the HTML canvas element with one pattern. 
[Deployment of Canvas Page](https://sewtable-canvaspage-erhyyesq1f.herokuapp.com/canvas)

### CODE REVIEW

File: components/canvas/drawingarea.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import styles from "./Drawing.module.css";
+import { useEffect } from "react";
+import { useRef } from "react";
+
+function Drawing() {
+  const canvasRef = useRef(null);
+
+  useEffect(() => {
+    const ctx = canvasRef.current.getContext("2d");
```
Comment (cboenke) [inline components/canvas/drawingarea.tsx:9]:
```suggestion
    const context = canvasRef.current.getContext("2d");
```

File: components/canvas/drawingarea.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    const image = new Image();
+    image.src = "/pattern_hoodie.svg";
+    image.onload = () => {
+      ctx.drawImage(image, 0, 0);
```
Comment (cboenke) [inline components/canvas/drawingarea.tsx:13]:
```suggestion
      context.drawImage(image, 0, 0);
```

File: components/canvas/drawingarea.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import styles from "./Drawing.module.css";
+import { useEffect } from "react";
+import { useRef } from "react";
```
Comment (mewiemers) [inline components/canvas/drawingarea.tsx:3]:
```suggestion
import { useEffect, useRef } from "react";
```
you only need one line in import

**Review by mewiemers — COMMENTED** [review]
GJ 🧵

**Review by cboenke — APPROVED** [review]
GJ! For consistency, you could change the component file names to Pascal case and for better readability the css class names to Camel case :)

**Review by RonjaMeinecke — APPROVED** [review]
✨good job

**Review by mhstrkmp — COMMENTED** [review]
lgtm 🚀

**Review by JuKahl — APPROVED** [review]
GJ 🚀

File: public/footer.svg [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 375 97" xmlns="http://www.w3.org/2000/svg">
+<defs>
+<clipPath id="a">
+<rect transform="translate(2443 1800)" width="375" height="97" fill="#fff" stroke="#707070" data-name="Rechteck 31"/>
+</clipPath>
+</defs>
+<g transform="translate(-2443 -1800)" clip-path="url(#a)" data-name="Gruppe maskieren 2">
+<rect transform="translate(2443 1801)" width="375" height="158" rx="35" fill="#4d4d4d" data-name="Rechteck 24"/>
```
Comment (HorstFrank) [inline public/footer.svg:9]:
```suggestion
<rect transform="translate(2443 1801)" width="375" height="158" rx="35" fill="#4d4d4d" />
```

You don't need all the ```data-name="XXX"``` attributes in your svgs.

**Review by HorstFrank — APPROVED** [review]
GJ

**Pull request merged**